2-api/cyf-ecommerce-api/server.js

```diff
@@ -60,42 +53,140 @@ app.post("/customers", function (req, res) {
     });
 });
 
-// GET ALL THE SUPPLIERS
-app.get("/suppliers", function (req, res) {
+// ADDING A NEW PRODUCT
+
+app.post("/products", function (req, res) {
+  let productName = req.body.product_name;
   pool
-    .query("SELECT * FROM suppliers")
+    .query(`SELECT * FROM products WHERE product_name = $1`, [productName])
```

Comment:
You want to make sure that the productName is in fact present in the request body, before you query